examples/navier-stokes/densitycurrent.h

```diff
@@ -78,15 +78,21 @@
 CEED_QFUNCTION(ICsDC)(void *ctx, CeedInt Q,
                       const CeedScalar *const *in, CeedScalar *const *out) {
 
+// *INDENT-OFF*
```

Comment:
Astyle tries to reverse indent this. The behavior of preproc linting inside of functions is apparently undefined and won't be fixed.